modelangelo/protocols/protocol_model_angelo.py

```diff
@@ -83,10 +80,53 @@ def _defineParams(self, form):
                        label="Use GPU for execution",
                        help="This protocol has both CPU and GPU implementation."
                             "Select the one you want to use.")
+                            
         form.addHidden(GPU_LIST, params.StringParam, default='0',
                        expertLevel=LEVEL_ADVANCED,
                        label="Choose GPU ID (single one)",
                        help="GPU device to be used")
+                       
```

Comment:
I'm guessing this points to a file, right?

In that case I'd use  FileParam?

Reply:
hi @pconesa,

 I guess you refer to "configFile" param rather than "GPU_LIST" param.

  configFile param has been modified from StringParam to FileParam as requested.


           form.addParam('configFile', params.FileParam,
                       label="Configuration File",
                       default="",
                       expertLevel=LEVEL_ADVANCED,
                       help="""this option is only for VERY advanced users,\n

Reply:
Yes, I was referring to configFile